tests/bsim/bluetooth/mesh/src/test_blob.c

```diff
@@ -149,7 +151,7 @@ static void blob_cli_caps(struct bt_mesh_blob_cli *b, const struct bt_mesh_blob_
 		ASSERT_EQUAL(caps->max_size, CONFIG_BT_MESH_BLOB_SIZE_MAX);
 		ASSERT_EQUAL(caps->min_block_size_log, BLOB_BLOCK_SIZE_LOG_MIN);
 		ASSERT_EQUAL(caps->max_block_size_log, BLOB_BLOCK_SIZE_LOG_MAX);
-		ASSERT_EQUAL(caps->max_chunk_size, BLOB_CHUNK_SIZE_MAX(BT_MESH_RX_SDU_MAX));
+		ASSERT_EQUAL(caps->max_chunk_size, MAX_CHUNK_SIZE);
```

Comment:
Please leave capability dependency on Rx SDU. Probably over new `BLOB_RX_CHUNK_SIZE`.

Reply:
If the added kconfigs aren't used in the tests, reverting it to ´BLOB_CHUNK_SIZE_MAX(BT_MESH_RX_SDU_MAX)´ should also be fine (especially given we do not touch the new kconfigs in the test setup, and if, we need other changes as well), I may even prefer it as it states clearer that we expect the max chunk size according to SDU.
Let me know if you prefer ´BLOB_CHUNK_SIZE_MAX(BT_MESH_RX_SDU_MAX)´ or `BLOB_RX_CHUNK_SIZE`
(FYI Ended up with the changes to the test, as if you change the chunk sizes, you need to amend the expectancies accordingly, which of course is not needed if we keep the test as is.)

Reply:
Ok then, please, leave `BLOB_CHUNK_SIZE_MAX(BT_MESH_RX_SDU_MAX)`